### DISCUSSION TITLE
Faire des set d'items

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1699] [Nouvelles idées]
Après avoir cherché dans toutes les suggestions, j'ai rien vu sur le sujet et ça m'a étonné alors voilà:
Faire des sets d'item (oui bon normalement vous avez lu le titre)
Les items que l'on pourrais associés comme la jambe mécanique et le bras mécanique pourrais bénéficier d'un petit bonus si tout deux équipés. Ce bonus serais appliqué seulement sur les stats que buff ou debuff les items du set et serais influencé par la rareté des items du set. Il est plus difficile de se procurer un set composé d'items légendaires qu'un set d'items peu commun, vous en conviendrez. Les items du set n'ont pas à être liés directement par le nom, exemple: un set sportif composé du bouclier de brennus, du ballon de fluxball (ou gants de boxe) et des skis glissants. Ou encore un set mythologique avec la foudre de Zeus et l'amulette de Midas. Enfin bref vous avez saisi l'idée (sauf si je me suis trop éparpillé).

**Comment by Pietagorh** [comment]
++

**Comment by FiFolker** [comment]
+1 franchement super idée, bizarre qu'elle n'ait pas été proposée avant mais franchement ça serait lourd et ça irait bien avec la maj PVP donnant différents intérêt à se stuff suivant la classe etc 'fin très bonne idée !

**Reply by Pietagorh** [reply]
J'l'avais proposé y'a trèèès longtemps j'crois, à un moment où on était même pas sur github pour les suggestions x)
Et depuis j'ai oublié :dumb:

**Reply by KirIcare** [reply]
Tu sais comment j'y ai pensé ? Bah j'ai deux items à faire avec mes dons donc je pensais faire 2 items liés et comme ça existe pas encore, je me suis dit "y'a qu'a le créer ptdr" 😂

**Comment by QQtin3** [comment]
Clairement pour, let's goooo
